Week3/joyeeon/Mission1/src/pages/DetailMoviePage.tsx

```diff
@@ -0,0 +1,59 @@
+import axios from "axios";
+import { Detail } from "../types/details";
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import DetailPanel from "../components/details/DetailPanel";
+import CastingList from "../components/details/CastingList";
+import IsLoading from "../components/state/IsLoading";
+import IsError from "../components/state/IsError";
+
+const DetailMoviePage = () =>{
+    const {id} = useParams();
+    const [detail, setDetail] = useState<Detail | null>(null);
+    const [error, setError] = useState<string | null>(null);
+    const token = import.meta.env.VITE_TMDB_TOKEN;
+    
+    useEffect(() =>{
+        const fetchDetailMovies = async () =>{
+            try {
+                const { data } = await axios.get(
+                    `https://api.themoviedb.org/3/movie/${id}?language=ko-KR`,
+                    {
+                        headers: {
+                            Authorization: token,
```

Comment:
OAuth 2.0 표준에 따라 토큰 타입을 명시하는 "Bearer"를 접두어로 붙여주세요!

```suggestion
                            Authorization: `Bearer ${token}`,
```